### PR TITLE
[hi] Update README-hi.md

### LINKED ISSUE #29519: [hi] Fix README-hi.md
Problem: 
The sentence below in the README-hi.md seems to be incomplete 

"अपने fork में कोई भी परिवर्तन करना चाहते हैं, और जब आप उन परिवर्तनों को हमारे पास भेजने के लिए तैयार हों, तो अपने fork पर जाएं और हमें इसके बारे में बताने के लिए एक नया pull request बनाएं।"
 the first part i.e "अपने fork में कोई भी परिवर्तन करना चाहते हैं" makes no proper connection with the other part

### PR DESCRIPTION
**Pull request by AvineshTripathi**
restructured a sentence 
fixed #29519

### CODE REVIEW

File: README-hi.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ## डॉक्स में योगदान देना
 
-आप अपने GitHub खाते में इस रिपॉजिटरी की एक copy बनाने के लिए स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इस copy को *Fork* कहा जाता है। अपने fork में कोई भी परिवर्तन करना चाहते हैं, और जब आप उन परिवर्तनों को हमारे पास भेजने के लिए तैयार हों, तो अपने fork पर जाएं और हमें इसके बारे में बताने के लिए एक नया pull request बनाएं।
+आप अपने GitHub खाते में इस रिपॉजिटरी की एक copy बनाने के लिए स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इस copy को *Fork* कहा जाता है। अपने fork के परिवर्तन करने के बाद जब आप उनको हमारे पास भेजने के लिए तैयार हों, तो अपने fork पर जाएं और हमें इसके बारे में बताने के लिए एक नया pull request बनाएं
```
Comment (anubha-v-ardhan) [inline README-hi.md:10]:
```suggestion
आप अपने GitHub खाते में इस रिपॉजिटरी की एक copy बनाने के लिए स्क्रीन के ऊपरी-दाएँ क्षेत्र में **Fork** बटन पर क्लिक करें। इस copy को *Fork* कहा जाता है। अपने fork में परिवर्तन करने के बाद जब आप उनको हमारे पास भेजने के लिए तैयार हों, तो अपने fork पर जाएं और हमें इसके बारे में बताने के लिए एक नया pull request बनाएं।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: README-hi.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply README-hi.md:10]: Grammatical mistakes and missing `।`

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM

**Comment by AvineshTripathi** [comment]
@divya-mohan0209 sure

**Comment by sftim** [comment]
It looks like you'd want to omit the second commit here @AvineshTripathi

**Comment by AvineshTripathi** [comment]
> It looks like you'd want to omit the second commit here @AvineshTripathi

yes if there is something still left to do please let me know @sftim

**Comment by sftim** [comment]
The squash _here_ is wrong because the second commit reverts the first commit.

**Comment by AvineshTripathi** [comment]
@sftim is it now fine ?

**Comment by divya-mohan0209** [comment]
@AvineshTripathi I can see 5 commits at this point on this PR. Please can you check the guide - https://kubernetes.io/docs/contribute/new-content/open-a-pr/ & squash + merge it correctly?

**Comment by AvineshTripathi** [comment]
@divya-mohan0209 I want the lastest commit to be merged and I am not able to remove the other 4 commits can you help me for that

**Comment by divya-mohan0209** [comment]
> @divya-mohan0209 I want the lastest commit to be merged and I am not able to remove the other 4 commits can you help me for that also I my git log I have removed the other 4 but still it is showing

You need to add, rebase, & then force push the content. Please check your Slack DM.

**Comment by anubha-v-ardhan** [comment]
@AvineshTripathi Are you still working on this PR. Can we have an update.

**Comment by AvineshTripathi** [comment]
@anubha-v-ardhan thank you for asking me I have sent you a message in slack do see it

**Comment by AvineshTripathi** [comment]
there were branch conflicts therefore had to close this opening a new PR for the same

**Pull request closed without merging**